docs/source/reference/skysim_reference.rst

```diff
@@ -0,0 +1,220 @@
+.. _skysim_reference:
+
+Array Telescope Simulation (``telsim``)
+========================================
+
+
+Table of Contents
```

Comment:
The things below do not link to anything even though you have created links to them. Either remove the links or link them to the relevant sections.

**same thing** for other "table of contents" you added